tools/analyse_query_usage.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Usage:
+#
+#   ./analyse_query_usage.sh my-file-of-urls.txt
+#
+#   cat my-file-of-urls.txt | ./analyse_query_usage.sh
+#
+#
+# For the input, a list of URLs, shows which are most common when you
+# ignore all query parameters. Eg, those that will be normalised in
+# mappings unless some query parameters are marked as important.
```

Comment:
For the same reason as above, could the second sentence here be something like 'the number of URLs that will be reduced to that single URL by canonicalization unless some query parameters are marked as significant'?